Gender Analysis.ipynb

```diff
@@ -1273,19 +1274,14 @@
     "\n",
     "        pickle.dump([a_indices, b_indices, scene_filepaths[i], a_probs, b_probs], open(\"results/{0}/4/{1}_info.pkl\".format(folder_name, names[categories[i]]), \"wb\"))\n",
     "\n",
-    "        np.random.shuffle(projected_features_scenes)\n",
-    "        clf_random.fit(projected_features_scenes, labels)\n",
-    "        acc_random = clf_random.score(projected_features_scenes, labels)\n",
-    "        value = acc / acc_random\n",
+    "        base_acc, rand_acc, p_value = permutation_test_score(clf, projected_features_scenes, labels, scoring=\"accuracy\", n_permutations=100)\n",
+    "        rand_acc = np.mean(rand_acc)\n",
     "        \n",
-    "        if value <= 1.2 and acc <= .7 and len(labels) < 25: # can tune as desired\n",
+    "        if p_value > 0.1: # can tune as desired\n",
```

Comment:
This can be updated if needed

Reply:
besides just p-value, can there be a test for strength of association as well? i.e., if p-value is very low but distributions are very different, that's not as important

Reply:
discussed over slack, but let's say if p_value < .05, and then based on your qualitative inspection, feel free to pick some acc or strength cut off that seems reasonable to go here